client/modules/IDE/pages/IDEView.jsx

```diff
@@ -427,6 +427,7 @@ class IDEView extends React.Component {
             <Feedback previousPath={this.props.ide.previousPath} />
           </Overlay>
         }
+        aa
```

Comment:
It looks like this was added by mistake?

Reply:
seems like it!